x-pack/plugins/cloud_security_posture/public/common/api/use_compliance_dashboard_data_api.ts

```diff
@@ -7,14 +7,36 @@
 
 import { type QueryObserverOptions, useQuery } from '@tanstack/react-query';
 import { useKibana } from '../hooks/use_kibana';
-import { ComplianceDashboardData } from '../../../common/types';
+import { PosturePolicyTemplate, ComplianceDashboardData } from '../../../common/types';
 import { STATS_ROUTE_PATH } from '../../../common/constants';
 
-const getStatsKey = ['csp_dashboard_stats'];
+const getCspmStatsKey = ['csp_cspm_dashboard_stats'];
+const getKspmStatsKey = ['csp_kspm_dashboard_stats'];
```

Comment:
maybe we can have 1 hook with a dynamic key: `['csp_stats', { policyTemplate}]` as it seems like `useCspmComplianceDashboardDataApi` and `useKspmComplianceDashboardDataApi` are identical

Reply:
yeah i like that idea. Admittedly, i just had an easier time mocking 2 hooks rather than one dynamic hook. i added a todo to make that change, we really need a day to clear some todos from our code by now anyway.